src/components/form.tsx

```diff
@@ -27,160 +28,195 @@ export default (props: {furniture?: Furniture, updateFurniture: (newFurniture: F
-      <textarea
-        id="standfirst"
-        name="standfirst"
-        placeholder="standfirst..."
-        value={props.furniture?.standfirst}
-        onChange={event => update('standfirst', event.target.value)}
-      ></textarea>
-
-      <fieldset>
-        <legend>Standfirst Size</legend>
-        <input
-          type="radio"
-          id="standfirstSmall"
-          name="standfirstSize"
-          value={StandfirstSize.Small}
-          checked={props.furniture?.standfirstSize === StandfirstSize.Small}
-          onChange={event => update('standfirstSize', event.target.value)}
-        />
-        <label htmlFor="standfirstSmall">Small</label>
-
-        <input
-          type="radio"
-          id="standfirstMedium"
-          name="standfirstSize"
-          value={StandfirstSize.Medium}
-          checked={props.furniture?.standfirstSize === StandfirstSize.Medium}
-          onChange={event => update('standfirstSize', event.target.value)}
-        />
-        <label htmlFor="standfirstMedium">Medium</label>
-      </fieldset>
-
-      <ColourPicker id="standfirst" colour={props.furniture?.standfirstColour} update={colour => update('standfirstColour', colour)}/>
-
-      <fieldset>
-        <legend>Position</legend>
-        <input
-          type="radio"
-          id="positionTop"
-          name="positionValue"
-          value="top"
-          checked={position == "top"}
-          onChange={event => updatePostion("top", 0)}
-        />
-        <label htmlFor="positionTop">Top</label>
-
-        <input
-          type="radio"
-          id="positionMiddle"
-          name="positionValue"
-          value="middle"
-          checked={position == "middle"}
-          onChange={event => updatePostion("middle", 40)}
-        />
-        <label htmlFor="positionMiddle">Middle</label>
-        <input
-          type="radio"
-          id="positionBottom"
-          name="positionValue"
-          value="bottom"
-          checked={position == "bottom"}
-          onChange={event => updatePostion("bottom", 100)}
-        />
-        <label htmlFor="positionBottom">Bottom</label>
-        <br/>
-        <div>
+        <Collapsible name="Headline">
+          <label htmlFor="headline">Text</label>
+          <textarea
+            id="headline"
+            name="headline"
+            placeholder="headline..."
+            value={props.furniture?.headline}
+            onChange={event => update('headline', event.target.value)}
+          />
+
+          <fieldset>
+            <legend>Size</legend>
+            <input
+              type="radio"
+              id="headlineSmall"
+              name="headlineSize"
+              value={HeadlineSize.Small}
+              checked={props.furniture?.headlineSize === HeadlineSize.Small}
+              onChange={event => update('headlineSize', event.target.value)}
+            />
+            <label htmlFor="headlineSmall">Small</label>
```

Comment:
There's some common code here between headline and standfirst. Is there any value in creating a component to represent this, similar to the colour picker? I'm not sure if there are plans to add more, similar inputs.

Reply:
It's a good point and one I have considered. I'm on the fence about it, I would be inclined to leave it as is and componentise in then future if it would be useful but I could be convinced either way if anyone has any strong feelings.